lms/djangoapps/edraak_university/admin.py

```diff
@@ -48,5 +47,17 @@ def edraak_profile(self, profile):
     def email(self, profile):
         return profile.user.email
 
+# UniversityIDSettings added to the admin panel
+class UniversityIDSettingAdmin(admin.ModelAdmin):
+    list_display=(
```

Comment:
- [ ] I see you're displaying all fields here, so you don't need to define `list_display`. Check [ModelAdmin.list_display](https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_display) documentation.